dashboard/app/views/layouts/_http_support.html.haml

```diff
@@ -9,18 +9,30 @@
     %i.fa.fa-warning
     = t("compatability.https.header")
   :javascript
+    function setCookie(key, value, days) {
```

Comment:
it would be possible, but I'm going to pass on introducing jQuery here for two reasons:

1. my general preference is to reduce our dependency on jQuery whenever possible rather than increase it
2. I'm trying to keep the URL-testing logic here roughly in sync with similar testing logic in [https_test](https://github.com/code-dot-org/code-dot-org/blob/staging/shared/middleware/https_test.html), which should have as small a footprint as possible in order to reduce the load time of the redirect page

In the future once the frontend pipeline stuff gets in a good place, I'd be all for making [js-cookie](https://github.com/js-cookie/js-cookie) available within our npm dependency graph and using that for a consistent cookie-manipulating interface.